From 4c289fafa170bb785d69f1c352fc828f33c6778a Mon Sep 17 00:00:00 2001
From: Allan McRae <allan@archlinux.org>
Date: Wed, 3 Apr 2024 09:35:09 +1000
Subject: [PATCH 29/N] libmakepkg: remove MAKEFLAGS from buildenv_vars

MAKEFLAGS should not be cleared by options=(!buildenv).

Regression introduced in commit 09e82f01ea824f9203b0ed7238d98b612669b1a9

Signed-off-by: Allan McRae <allan@archlinux.org>
---
 scripts/libmakepkg/buildenv.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/libmakepkg/buildenv.sh.in b/scripts/libmakepkg/buildenv.sh.in
index 031ce49..830664b 100644
--- a/scripts/libmakepkg/buildenv.sh.in
+++ b/scripts/libmakepkg/buildenv.sh.in
@@ -25,7 +25,7 @@ LIBMAKEPKG_BUILDENV_SH=1
 MAKEPKG_LIBRARY=${MAKEPKG_LIBRARY:-'@libmakepkgdir@'}
 
 declare -a buildenv_functions build_options
-buildenv_vars=('CPPFLAGS' 'CFLAGS' 'CXXFLAGS' 'LDFLAGS' 'MAKEFLAGS' 'CHOST')
+buildenv_vars=('CPPFLAGS' 'CFLAGS' 'CXXFLAGS' 'LDFLAGS' 'CHOST')
 
 for lib in "$MAKEPKG_LIBRARY/buildenv/"*.sh; do
 	source "$lib"
